h2o-core/src/main/java/water/H2ONode.java

```diff
@@ -53,12 +53,12 @@ public void stopSendThread(){
   /** Identification of the node via IP and PORT.
    *
    */
-  static final class H2Okey extends InetSocketAddress implements Comparable {
+  public static final class H2Okey extends InetSocketAddress implements Comparable {
```

Comment:
This was intentionally private -  i would still keep it private since it can bring confusion with `water.Key`